sample/src/main/java/io/polygon/kotlin/sdk/sample/KotlinUsageSample.kt

```diff
@@ -364,4 +361,9 @@ fun getQuotesSample(polygonClient: PolygonRestClient) {
     )
     polygonClient.getQuotesBlocking(params).pp()
 
+}
+
+fun getLastTradeV2Sample(polygonClient: PolygonRestClient) {
+    println("Last F Trade")
+    polygonClient.stocksClient.getLastTradeBlockingV2("F").pp()
```

Comment:
Is there a sample in here for the deprecated last trade? If so let's replace that instead of making a new sample

Reply:
There is, replaced it